HousingRepairsSchedulingApi/Services/Drs/DrsService.cs

```diff
@@ -184,6 +184,27 @@ public async Task<order> SelectOrder(string bookingReference)
             return drsOrder;
         }
 
+        public async Task DeleteBooking(string bookingReference, int bookingId)
+        {
+            Guard.Against.NullOrWhiteSpace(bookingReference, nameof(bookingReference));
+
```

Comment:
Do we need any validation on the bookingId, i.e. > 0?

Reply:
Although it's unlikely DRS booking Ids allow negative values, without knowing for certain, I decided to omit validation.

Validation could be added quite easily though.
Should we make an assumption they are positive values and add validation?

Reply:
Perhaps it's overkill to check for negative values, as you say we can't be sure that booking ids can't be negative.